java/src/jmri/jmrit/operations/rollingstock/cars/JmritOperationsCarsBundle_de.properties

```diff
@@ -2,7 +2,172 @@
+carAddColor=Wagenfarbe hinzuf\u00fcgen?
+ownerNameNotExist=Eignername "{0}" besteht nicht in Betrieb; willst du ihn hinzuf\u00fcgen?
+addOwner=Eigner hinzuf\u00fcgen?
+loadNameNotExist=Ladung "{0}" besteht nicht in Betrieb; willst du ihn hinzuf\u00fcgen?
+addLoad=Ladung hinzuf\u00fcgen?
+NameResChar=Eingabe darf keine vorbehaltene Buchstaben enthalten
+carTrainNotService=Wagen ({0}) kann nicht aufgenommen werden von Zug ({1})
+carLengthMustBe=Wagenl\u00e4nge soll ein Ganzzahl in Fu\u00df sein
+findCar=Suche Wagen nach Bahnnummer. Stern "*" = Stellvertretersymbol.
+carWithRoadNumNotFound=Wagen mit Bahnnummer "{0}" nicht angetroffen
+carCouldNotFind= Wagen nicht gefunden!
+carSureDelete=Bist du sicher du willst alle Wagen vom Betrieb l\u00f6schen?
+carDeleteAll=Alle Wagen l\u00f6schen?
+carSureResetMoves=Bist du sicher du willst das Schrittzahl aller Wagen in Betrieb zur\u00fcckstellen auf 0?
+carResetMovesAll=Schrittzahl zur\u00fcckstellen?
+carDeleteCarsTrack=Bist du sicher du willst alle Wagen vom Betrieb l\u00f6schen?
+rsInRoute=Wagen hat ein Ziel und Zug bekommen
+LastMoved=Letzter Schritt
+Replace=Ersetze
+
+carRoadNum=Bahnnumer des Wagens kann nur {0} Ziffern lang sein
+roadNumNG=Kann Bahnnummer nicht nutzen!
+
+# car set frame messages
+rsType=Wagen
+rsFullySelect=Um ein Wagen zu setzen, bitte sein Ort und Gleis ausw\u00e4hlen
+rsNotMove=Wagen kann keine Schritt machen!
+rsCanNotDest=Kann Wagenziel nicht aktualisieren
+carPartKernel=Wagen ist Teil der Wagengruppe ({0})
+
+# cars set frame messages
+carNoneSelected=Keine Wagen ausgew\u00e4hlt!
+PriorityLow=Niedrig
+PriorityHigh=Hoch
+ButtonDontShow=Nicht mehr zeigen
+
+# car import
+ImportCars=Importiere Wagen
+LineNumber=Zeile Nummer:
+
+# tool tips
+
+# tool menu items
+MenuDeleteCarsTrack=L\u00f6sche Wagen auf Gleis {0}
+MenuEnableDestination=Aktiviere Ziel
+MenuItemResetMoves=Arbeitszahl f\u00fcr alle Wagen zur\u00fcckstellen auf 0?
+MenuItemDelete=L\u00f6sche alle Wagen
+
+# print options
+PrintOptions=Druckoptionen
+PrintCarsWithLocation=Drucke nur Wagen mit Ort
+PrintCarLength=Drucke Wagenl\u00e4nge
+PrintCarWeight=Drucke Wagengewicht
+PrintCarColor=Drucke Wagenfarbe
+PrintCarOwner=Drucke Wageneigner
+PrintCarBuilt=Drucke Wagenbaujahr
+PrintCarLastMoved=Drucke letzte Wagenschritt
+PrintCarWait=Drucke Wartezahl des Wagens
+PrintCarPickup=Drucke Abholplanung des Wagens
+PrintCarLoad=Drucke Wagenladung
+PrintCarLocation=Drucke Wagenort
+PrintCarTrain=Drucke Zug des Wagens
+PrintCarDestination=Drucke Wagenziel
+PrintCarFinalDestination=Drucke Endziel des Wagens
+PrintCarReturnWhenEmpty=Drucke R\u00fcckfahrt Wenn Leer des Wagens
+PrintCarComment=Drucke Wagenbemerkung
+
+# print car loads
+LoadDropMessage=Lade Zustellwagenbericht
+LoadPickupMessage=Lade Abholwagenbericht
+
+# Export Cars
+ExportedCarsToFile=Exportiere {0} Wagen nach Datei: {1}
+
+carColors=Schwartz,Blau,Braun,Grau,Gelb,Gr\u00fcn,Orange,Violett,Rot,Silber,Tuskanrot,Wei\u00df
```

Comment:
@DanielBoudreau this translation is not picked up in the interface drop down list, nor is Loads/Empties (L/E). Does that mean it's useless to add them, or could we try to make this work?